katpoint/test/test_flux.py

```diff
@@ -17,7 +17,12 @@
 """Tests for the flux module."""
 from __future__ import print_function, division, absolute_import
 
-import unittest2 as unittest
+import sys
+
+if sys.version_info < (3,):
```

Comment:
Ah right, the commit message should have read "Python 2" :-)

Reply:
Oops. I'll reword it before merging.